openshift_images/using_images/images-other-jenkins.adoc

```diff
@@ -0,0 +1,85 @@
+[id="images-other-jenkins"]
+= Configuring Jenkins images
+include::modules/common-attributes.adoc[]
+:context: images-other-jenkins
+toc::[]
+
+{product-title} provides a container image for running Jenkins. This image
+provides a Jenkins server instance, which can be used to set up a basic flow for
+continuous testing, integration, and delivery.
+
+This image also includes a sample Jenkins job, which triggers a new build of a
+BuildConfig that is defined in {product-title}, tests the output of that build,
+and then on successful build, retags the output to indicate that the build is ready
+for production.
+
+{product-title} follows the link:https://jenkins.io/changelog-stable/[LTS]
+release of Jenkins. {product-title} provides an image that contains Jenkins 2.x.
+
+The {product-title} Jenkins agent images come in two varieties, Red Hat
+Enterprise Linux (RHEL) 7 based images and CentOS 7 Based Images.
+
+[discrete]
+=== RHEL 7 Based Images
+
+RHEL 7 Jenkins images are available through the Red Hat Registry:
+
+----
+$ docker pull quay.io/openshift4/jenkins-2-rhel7
```

Comment:
No this image in quay.io.
In the registry.redhat.io regsitry, jenkins images for 4.x:
regsitry.redhat.io/openshift4/ose-jenkins:<v4.1.4>
regsitry.redhat.io/openshift4/ose-jenkins-agent-nodejs:<v4.1.4>
regsitry.redhat.io/openshift4/ose-jenkins-agent-maven:<v4.1.4>
regsitry.redhat.io/openshift4/ose-jenkins-agent-base:<v4.1.4>
In quay.io, the jenkins images  are stored with digest numbers like this:
$oc adm release info --pullspecs quay.io/openshift-release-dev/ocp-release:4.1.4  | grep  jenkins 
  jenkins                                       quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:bec7f571b6ad293e697206786deb8194cd67d6bedf8e7eddb24290838807272a
  jenkins-agent-maven                           quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:20e18317a9284549f56814d720da33393946f8f6add15f56974015d7abb78ba1
  jenkins-agent-nodejs                          quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:98f342ba0aa7f6d012bf3f03d4a182aa92a1563ded39671b8c13f825e1348978